mcp/src/ui/global.css

```
@@ -0,0 +1,35 @@
+
+button,
+input,
+textarea,
+select {
+  font: inherit;
+}
+
+code,
+pre {
+  font-family:
+    "SFMono-Regular", "SF Mono", ui-monospace, Menlo, Consolas, monospace;
```

Stylelint (17.9.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)